### PR TITLE
feat(tsc): remove webpack, add tsc

### PR DESCRIPTION
**Pull request by vijaypemmaraju**
Goodbye webpack. More importantly, this PR also adds a type declaration file, which is important for the typings to work wherever we install this package.

### CODE REVIEW

File: build/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+declare module '@groove-labs/action-utils' {
```
Comment (corytheboyd) [inline build/index.d.ts]: I wonder if we should declare a namespace too like `ActionUtils`

Comment (vijaypemmaraju) [reply build/index.d.ts]: that would be a breaking change, right?

Comment (vijaypemmaraju) [reply build/index.d.ts]: reading: http://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html#needless-namespacing

Comment (corytheboyd) [reply build/index.d.ts]: gotcha, sounds good then! and yeah based on that looks like it would indeed be a breaking change

**Comment by corytheboyd** [comment]
image

I could literally cry right now

**Pull request merged**